packages/project-config/prettier/base.js

```diff
@@ -0,0 +1,25 @@
+// @ts-check
+
+/** @type {import('@trivago/prettier-plugin-sort-imports').PrettierConfig} */
+const config = {
+  singleQuote: true,
+  semi: false,
+  printWidth: 100,
+  tabWidth: 2,
+  trailingComma: 'es5',
+  plugins: ['@trivago/prettier-plugin-sort-imports'],
+  importOrder: [
+    '^@nestjs',
+    '^react',
+    '<THIRD_PARTY_MODULES>',
+    '^@(admin-api|admin-web|api|web)',
```

Comment:
there is `@reg-scraper` app too, no?

Reply:
fixed

Reply:
Can we use something like `^@\w`? Or would it match 3p modules that begin with @ too?

Reply:
yes, there will be @mui @apollo etc.

Reply:
I suggest that we should call all local absolute path as @/ instead of @<app-name>/. Shorter import path and easier to define the import order rule.